modern-business/style-editor.scss

```diff
@@ -21,7 +43,7 @@ body {
 
 	@include media(mobile) {
 
-		.wp-block[data-align="full"] {
+		#{get-block-selector("full")} {
```

Comment:
This syntax looks a bit awkward to me. Wouldn't a mixin be a better fit for this? Or is it just me not being used to use functions like this?

Reply:
Yeah that's a great question. The goal here was just to reduce the amount of verbosity on this line - otherwise it would have several lines of selectors. And since those selectors are repeated so often, I figured it might be worthwhile to see what a function could do in scss. I wasn't sure if a mixin would do the same thing. Do those basically allow you to use parameters to create reusable styles? In this case I wanted to do that with a selector instead of a style.

Reply:
> Do those basically allow you to use parameters to create reusable styles?

You can also use them to create reusable selectors and inject the given styles with `@content`. This is actually how the mixins used for the responsive breakpoints work:

```
@function get-block-selector( $alignment ) {
	// ...
}

@mixin blocks($alignment) {
	#{get-block-selector( $alignment )} {
		@content;
	}
}

body {
	@include media(mobile) {
		@include blocks("full") {
			width: calc( 100% + 90px );
			max-width: calc( 100% + 90px );
		}
	}
}
```

I found that code more legible, but as I said maybe it's just me not being used to use functions on selectors :) So feel free to disregard this idea!

Reply:
Neat! I feel like it adds a bit of extra boilerplate, but I can definitely see how it's easier to read inline